### ISSUE TITLE
having problems

### ISSUE THREAD

**Issue by petitemuse** [darickc/MMM-BackgroundSlideshow#6]
Hello,

First I want to say that is really a great platform with a lot possibilities. 

5 days ago I bought this raspberry 3b out of curiosity: I wanted to do a Magic Mirror!

Well it is working! only after following all the steps of this module I am not managing to make it work (ok that was not part of the the Mirror project since it will be more a wall monitor)

so in the config.js I have written (copy paste + some changes since if I copied the original it was giving me Syntax error)

       {
                        module: "MMM-BackgroundSlideshow",
                        position: "fullscreen_below",
                        config: {
                                imagePaths: ["MMM-BackgroundSlideshow/example1"],
                                transitionImages: true,
                                randomizeImageOrder: true
                        }
                },
now the Mirror works again but with no images. Since I am using a SHH thing (lol, sorry for my ignorance!) I could see the following message:

Whoops! There was an uncaught exception...
{ Error: ENOENT: no such file or directory, scandir 'MagicMirror/modules/MMM-BackgroundSlideshow/example1'
    at Object.fs.readdirSync (fs.js:904:18)
    at Object.fs.readdirSync (ELECTRON_ASAR.js:608:28)
    at Class.gatherImageList (/home/pi/MagicMirror/modules/MMM-BackgroundSlideshow/node_helper.js:62:59)
    at Class.socketNotificationReceived (/home/pi/MagicMirror/modules/MMM-BackgroundSlideshow/node_helper.js:121:28)
    at Socket.<anonymous> (/home/pi/MagicMirror/modules/node_modules/node_helper/index.js:113:11)
    at emitTwo (events.js:126:13)
    at Socket.emit (events.js:214:7)
    at /home/pi/MagicMirror/node_modules/socket.io/lib/socket.js:528:12
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
  errno: -2,
  code: 'ENOENT',
  syscall: 'scandir',
  path: 'MMM-BackgroundSlideshow/example1' }

Now the file with 4 pictures is existing in MMM-BackgroundSlideshow/example1 (well I did change the file name to example1

what can I do that is work?

thank you so much

**Comment by darickc** [comment]
It needs to be like this:
`imagePaths: ["modules/MMM-BackgroundSlideshow/example1"],`

**Comment by petitemuse** [comment]
thank you it is now working! that looks amazing!

Never the less I have 2 questions:
 What should I write to have the pictures longer on the screen  

and second, and I think it will not be possible: I have a picture that is more on white, I can't see the informations of the weather for exemple. Is there any way to have the modules more clear?

anyway thank you!!

**Comment by darickc** [comment]
You would use slideshowSpeed for them to be longer.  It is in milliseconds, so 1000 = 1 second.  
You will have to play with the gradient settings to get it how you like.  Here is what I have mine at:

gradient: [
          'rgba(0, 0, 0, 0.75) 0%',
          'rgba(0, 0, 0, 0) 40%',
          'rgba(0, 0, 0, 0) 70%',
          'rgba(0, 0, 0, 0.75) 100%'
        ],

One way to see it easier is to set the background to completely white.  Then tweak the gradient to your liking.

**Comment by petitemuse** [comment]
thank you darickc !!

it did work!

Only: I do not know how to read the radiant bracket:
rgba: red green blue ... what is the a for?

why is there 4 lines and what is the % meaning?

thank you for your patience towards a niewbe!!!

**Comment by darickc** [comment]
a is for Alpha or the opacity.  It is decimal value between 0 and 1.  0 is transparent and 1 is opaque.  The percent of the four lines refers to the position on the screen.  0% is the top of the screen, 100% is the bottom (if going vertically, there is now an option for the gradient to be horizontal as well).  You aren't limited to four lines either.  You can have as many as you want.  I use four lines to make it easier to read text on the top and bottom of the screen.  If you wanted just the top line you could do something like this:
gradient: [
'rgba(0, 0, 0, 0.75) 0%',
'rgba(0, 0, 0, 0) 40%'
],
